scripts/benchmark-analyze.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from benchalerts import update_github_status_based_on_regressions
+
+
+Z_SCORE_THRESHOLD = 500
+
+# CIRCLE_* env vars are documented here:
+# https://circleci.com/docs/env-vars#built-in-environment-variables
+# This script also requires the GITHUB_API_TOKEN and CONBENCH_URL env vars.
```

Comment:
To enable this behavior, an admin will have to add a `GITHUB_API_TOKEN` env var to the CircleCI job with at least the `repo:status` permission scope. This will allow us to post a "conbench" github status to PR commits that says whether there were any severe [regressions](https://github.com/conbench/benchalerts/blob/main/benchalerts/workflows.py#L30).

Reply:
Can this be done by editing the circle ci / config ? I can try and do this , if it cant be done as a part of this PR.

Reply:
I don't think I can as part of this PR. For example, the current jobs are set up with the [CONBENCH_EMAIL](https://app.circleci.com/pipelines/github/facebookincubator/velox/13209/workflows/846da582-e063-46a4-9789-38c51574bffb/jobs/79262?invite=true#step-99-29) environment variable, but the only time that appears in this repo is when it's used [here](https://github.com/facebookincubator/velox/blob/main/.circleci/config.yml#L342). Someone must have set that up in the project settings.

Reply:
Ok, I think I know where to set that up. Basic question; How do I get the value for the api token ?

Reply:
You can set up a personal access token with just the `repo:status` permission scope following [these](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) directions. If you have access to a "bot" user that has access to this repo, or an already-existing PAT somewhere, that would be even better.

This won't be a long-term solution; the intent is that we'll set up a [GitHub App](https://docs.github.com/en/developers/apps/getting-started-with-apps/about-apps) that can be installed here instead. That will allow us to post rich-text results instead of a simple check.

Reply:
The other env variables I set up directly on circleCI UI. Let me ping you offline to get the actual value we need there.

Reply:
@pedroerp set up the environment variable, so it's working now!

Note that the icon is Pedro's profile picture. 😅 Unfortunately we can't change this, but it will also be fixed when we turn the status check into a GitHub App. That will come soon.